tests/fuzzer.c

```diff
@@ -1960,6 +1960,34 @@ static int basicUnitTests(U32 const seed, double compressibility)
         DISPLAYLEVEL(3, "OK \n");
     }
 
+    {
+        ZSTD_CCtx* cctx = ZSTD_createCCtx();
+        size_t zerosLength = ZSTD_BLOCKSIZE_MAX * 2 - 1;
+        size_t expectedOffsets[] = {1, 1};
+        size_t expectedLitLengths[] = {2, 1};
+        size_t expectedMatchLengths[] = {ZSTD_BLOCKSIZE_MAX - 2, ZSTD_BLOCKSIZE_MAX - 2};
+        size_t expectedReps[] = {1, 1};
+        size_t expectedMatchPos[] = {2, 1};
+        size_t expectedSequencesSize = 2;
+        size_t sequencesSize;
+        size_t i = 0;
+        ZSTD_Sequence* sequences = (ZSTD_Sequence*)compressedBuffer;
+        DISPLAYLEVEL(3, "test%3i : ZSTD_getSequences zeros : ", testNb++);
+        assert(cctx != NULL);
+        memset(CNBuffer, 0, zerosLength);
```

Comment:
Instead of compressing only zeros, can we have a more realistic scenario?

It should be easy to write a function

```c
size_t decompressSequences(BYTE* dst, size_t dstCapacity, BYTE const* src, size_t srcSize, ZSTD_Sequence const* seqs, size_t nbSeq);
```

since the sequences tell you exactly what to do. Then you can call `ZSTD_getSequences()` on `CNBuffer` and validate that `decompressSequences()` works.